listen/property-changes.js

```diff
@@ -320,6 +350,61 @@ PropertyChanges.prototype.makePropertyObservable = function (key) {
             configurable: true
         };
     } else { // 'get' or 'set', but not necessarily both
+		var setter,
+            getter = overriddenDescriptor.get ? function() {return overriddenDescriptor.get.apply(this, arguments);} : null;
```

Comment:
Please wrap deeply nested expressions such that they do not exceed 80 characters. This is a case where a ternary operator is not better than an if/else block.